src/components/ReadingGoal/UpdateReadingGoalModal/index.tsx

```diff
@@ -105,11 +125,11 @@ const UpdateReadingGoalModal = ({ isDisabled, readingGoal }: UpdateReadingGoalBu
       type,
       amount,
     };
-    if (duration) data.duration = duration;
+    if (isContinuous) data.duration = duration;
```

Comment:
nit: We should add curly brackets to `if` statements to make it consistent with the rest of the codebase. Later will add a linting rule